content/en/references/coverage/index.md

```diff
@@ -113,20 +113,20 @@ The table is a work in progress and will be updated as more functions are implem
 |BUILD_SCOPED_FILE_URL|❓|❓|
 |BUILD_STAGE_FILE_URL|❓|❓|
 |CASE|❓|❓|
-|CAST , ::|❓|❓|
+|CAST|❓|❓|
```

Comment:
I think `, ::` is left to suggest `::` as the other alternative to `CAST`:
https://docs.snowflake.com/en/sql-reference/functions/cast

But I may agree to put it as a code format to look better maybe?